src/server/pkg/hashtree/hashtree.go

```diff
@@ -131,7 +131,7 @@ func list(fs map[string]*NodeProto, path string) ([]*NodeProto, error) {
 	for i, child := range d.Children {
 		result[i], ok = fs[join(path, child)]
 		if !ok {
-			return nil, errorf(Internal, "could not find node for the child \"%s\" "+
+			return nil, errorf(Internal, "could not find file for the child \"%s\" "+
```

Comment:
"child" isn't a term we really use in general?  Can that be changed to something more meaningful?

Reply:
If you think there's a more meaningful term tell me what it is. Child is the most standard term I know for this.

Reply:
I'll rephrase, what is "child" in this context. I don't know what that's referring to. It's totally possible for this, and my other comments, that I'm just not familiar with normal file system terms, but figured I'd ask

Reply:
The files contained within a directory are referred to as "children" of the directory.